boa_engine/src/vm/opcode/mod.rs

```diff
@@ -126,32 +126,32 @@ macro_rules! generate_impl {
                 unsafe { std::mem::transmute(value) }
             }
 
+            const STR_MAP: &[&'static str] = &[
```

Comment:
Nitpick: The `MAP` suffix is a bit redundant on every variable, can you rename to `NAMES`, `INSTRUCTIONS` and `EXECUTE_FNS`?